pkg/cli/update.go

```diff
@@ -0,0 +1,151 @@
+package cli
+
+import (
+	"archive/zip"
+	"context"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"syscall"
+
+	"github.com/google/go-github/github"
+	"github.com/sosedoff/pgweb/pkg/command"
+)
+
+var (
+	latestVersion string
+	workDir       = "/tmp"
+	distName      = fmt.Sprintf("pgweb_%s_%s", runtime.GOOS, runtime.GOARCH)
+	zipName       = distName + ".zip"
+)
+
+func checkUpdate() {
+
+	if options.DisableCheckUpdate {
+		return
+	}
+
+	client := github.NewClient(nil)
+
+	latestRelease, _, err := client.Repositories.GetLatestRelease(context.Background(), "sosedoff", "pgweb")
+	if err != nil {
+		fmt.Errorf("GitHub is unavailable")
+	}
+
+	latestVersion = *latestRelease.Name
+
+	if latestVersion != command.VERSION {
+		var upgrade string
+
+		fmt.Printf("A new version %s is available. Do you want to upgrade? [y/N]", latestVersion)
+		chars, err := fmt.Scanln(&upgrade)
+		if err != nil || (chars > 0 && upgrade == "y") {
+			fmt.Println("Downloading and installing new version...")
+			installUpdate()
+		} else {
+			return
+		}
+
+	}
+
+}
+
+func installUpdate() {
+	var platformDownloadUrl = fmt.Sprintf("https://github.com/sosedoff/pgweb/releases/download/%s/v%s",
```

Comment:
URL format here is wrong, it should be `/releases/download/v%s/%s`.
I tried running the auto-update and it panic'ed because of the wrong url.

Reply:
You right, I hurried and made the typo!

Reply:
Fixed